### DISCUSSION TITLE
Use alpine with slugify

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [alpinejs/alpine#1935] [1. Help] [answered]
I tried to turn a string into a slug and output it in the dom using slugify, but it didn't work, here is my code below:
https://jsfiddle.net/ofcsLmvj/
I can see the original string but the slug didn't show up as expected

**Comment by KevinBatdorf** [comment, marked as answer]
You can't use `require()` in the browser like that. Check the browser console and you will see the error message.

**Reply by Chandler-Zhu** [reply]
Is there another way to implement this ? I was just following the steps in their docs

**Reply by KevinBatdorf** [reply]
You need a bundler like Webpack or something similar. Modern browsers support imports though. You could do this:

https://codepen.io/KevinBatdorf/pen/xxdMbOz

```js
import slugify from 'https://cdn.skypack.dev/@sindresorhus/slugify';
import Alpine from 'https://cdn.skypack.dev/alpinejs';
document.addEventListener('alpine:init', () => {
  Alpine.data('thething', () => ({
    str: 'This is a string',
    slug: '',
    init() {
    	this.slug = slugify(this.str)
    }  
  }))
})
Alpine.start()
```
